### PR TITLE
fix #74014 pinayrare.net

### LINKED ISSUE #74014: pinayrare.net
### Issue URL (Anti Adblock Script)
 [https://pinayrare.net/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fpinayrare.net%2F)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard for Windows v7.6 nightly 14
Browser: | Chrome
AdGuard driver: | WFP
Stealth mode options: | Hide your search queries, <br>Send Do-Not-Track header, <br>Remove X-Client-Data header from HTTP requests, <br>Strip URLs from tracking parameters, <br>Self-destructing third-party cookies (180), <br>Block Push API, <br>Block Location API
DNS filtering: | server: `https://dns-unfiltered.adguard.com/dns-query`<br>filters: `User rules`
Filters: | <b>Ad Blocking:</b><br/> EasyList, <br/>Peter Lowe's Blocklist, <br/>AdGuard Mobile Ads, <br/>AdGuard Base<br/><br/><b>Privacy:</b><br/> EasyPrivacy, <br/>AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> Adblock Warning Removal List, <br/>I don't care about cookies, <br/>EasyList Cookie List, <br/>AdGuard Annoyances<br/><br/><b>Security:</b><br/> Online Malicious URL Blocklist, <br/>Spam404, <br/>NoCoin Filter List<br/><br/><b>Language-specific:</b><br/> Frellwit's Swedish Filter
Custom filter subscriptions: | https://raw.githubusercontent.com/AdguardTeam/FiltersRegistry/master/filters/filter_15_DnsFilter/filter.txt

### PR DESCRIPTION
**Pull request by wout-codes**
#74014

### CODE REVIEW

File: EnglishFilter/sections/antiadblock.txt [hunk 1]
```diff
@@ -544,6 +544,9 @@ kzclip.com,ru-clip.com,ruclip.com###morev
 !
 !
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/74014
+||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$script,xmlhttprequest,redirect=nooptext,domain=pinayrare.net
+@@||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$domain=pinayrare.net
```
Comment (zloyden) [inline EnglishFilter/sections/antiadblock.txt:549]: It should be like the example below
```
! https://github.com/AdguardTeam/AdguardFilters/issues/74014
@@||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$domain=pinayrare.net
||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$script,redirect=googlesyndication-adsbygoogle,important,domain=pinayrare.net
```

**Review by zloyden — CHANGES_REQUESTED** [review]

File: EnglishFilter/sections/antiadblock.txt (hunk 1 above)
Comment (wout-codes) [reply EnglishFilter/sections/antiadblock.txt:549]: What's the difference @zloyden ?

**Review by zloyden — APPROVED** [review]

**Comment by zloyden** [comment]
Thanks!

File: EnglishFilter/sections/antiadblock.txt (hunk 1 above)
Comment (zloyden) [reply EnglishFilter/sections/antiadblock.txt]: This is necessary for our compiler and we have this order of rules.
Also `$script,redirect=googlesyndication-adsbygoogle` is enough for redirect rule, but you've missed the `important` modificator for this one.

Comment (wout-codes) [reply EnglishFilter/sections/antiadblock.txt]: Ok.

**Pull request merged**